doc: fix Sphinx warnings on Travis-CI

kenhys opened the pull request:
This PR fixes the following warnings:

  /home/travis/build/groonga/groonga/doc/source/news.rst:194: WARNING: unknown document: /reference/window_functions
  /home/travis/build/groonga/groonga/doc/source/reference/configuration.rst:46: WARNING: unknown document: reference/commands/config_delete
  /home/travis/build/groonga/groonga/doc/source/reference/configuration.rst:47: WARNING: unknown document: reference/commands/config_get
  /home/travis/build/groonga/groonga/doc/source/reference/configuration.rst:48: WARNING: unknown document: reference/commands/config_set
  /home/travis/build/groonga/groonga/doc/source/news.rst:68: WARNING: inconsistent term references in translated message. original: [], translated: [':doc:`/reference/functions/highlight_html`', ':doc:`reference/functions/snippet_html`']

kenhys committed:
doc: fix a typo

kenhys committed:
doc: fix missing slash in :doc: markup

kenhys committed:
doc: update po

kou commented on doc/locale/ja/LC_MESSAGES/news.po:
```diff
@@ -153,10 +153,7 @@ msgid ""
-"[:doc:`/reference/functions/highlight_html`], [:doc:`reference/functions/"
-"snippet_html`] slicesパラメータ指定時に、slices実行前の式からもキーワードを抽"
-"出するように改良しました。"
+msgstr "slicesパラメータ指定時に、slices実行前の式からもキーワードを抽出するように改良しました。"
```
Why the following is removed? It's needed content.

```text
[:doc:`/reference/functions/highlight_html`], [:doc:`reference/functions/snippet_html`]
```

kenhys replied:
I've edited wrong news entry. The next entry should be modified.

kenhys committed:
doc: fix a typo

kenhys committed:
doc: update po

kenhys committed:
doc: update po

This commit fixes :doc: mismatch.:

   WARNING: inconsistent term references in translated message. original:
   [], translated: [':doc:`/reference/functions/highlight_html`',
   ':doc:`reference/functions/snippet_html`']
